internal/configs/configmaps.go

```diff
@@ -499,6 +499,15 @@ func ParseConfigMap(cfgm *v1.ConfigMap, nginxPlus bool, hasAppProtect bool, hasA
 				glog.Error("ConfigMap Key 'app-protect-physical-memory-thresholds' must follow pattern: 'high=<0 - 100> low=<0 - 100>'. Ignoring.")
 			}
 		}
+		if appProtectReconnectPeriod, exists := cfgm.Data["app-protect-reconnect-period-seconds"]; exists {
+			if period, err := ParseFloat64(appProtectReconnectPeriod); err == nil {
+				if period > 0 && period <= 60 {
+					cfgParams.MainAppProtectReconnectPeriod = appProtectReconnectPeriod
+				}
+			} else {
+				glog.Error("ConfigMap Key 'app-protect-reconnect-period-second' must have value between '0' and '60'. '0' is illegal. Ignoring.")
```

Comment:
I find this a little bit confusing. If the value is between `0` and `60` I would assume that `0` and `60` are included. Is `60` a valid value?
Can you also add the valid values to the configmap key description?

Reply:
Hi. the same here, its up to doc "Value Range: 0-60. 0 is illegal". so 60 is valid and 0 is not.

Reply:
I don't think it hurts to specify the values in the description and to make it more clear even if it doesn't match the one on the website.

Reply:
added some wording, please reopen if needed